guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -38,6 +38,8 @@ endif::[]
 
 For detailed information on partitioning and size, refer to the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[{RHEL} 8 partitioning guide].
 
+For external database servers: `/var/lib/pgsql` with installation size of 100 MB and runtime size of 20 GB.
+
```

Comment:
```suggestion
For external database servers: `/var/lib/pgsql` with installation size of 100 MB and runtime size of 20 GB.

For detailed information on partitioning and size, refer to the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[{RHEL} 8 partitioning guide].

```
I apologize I didn't catch this sooner but I think the requirement should be placed before the reference to another guide. Any thoughts on this?